bodhi/tests/server/services/test_updates.py

```diff
@@ -3080,8 +3090,8 @@ def test_manually_push_critical_update_with_autopush_turned_off(self, publish, *
 
     @mock.patch(**mock_valid_requirements)
     @mock.patch('bodhi.server.notifications.publish')
-    def test_autopush_non_critical_update_with_negative_karma(self, publish, *args):
-        """Autopush Non Critical update even though it receives negative karma"""
+    def test_disable_autopush_non_critical_update_with_negative_karma(self, publish, *args):
+        """Disable Autopush Non Critical update even though it receives negative karma"""
```

Comment:
The wording of the end of this sentence is a little confusing since it's from the way the comment used to be. How about "Disable autokarma on non-critical updates upon negative comment."